server/database/seed.js

```diff
@@ -56,4 +56,5 @@ const seedManyUsers = (start, number) => {
     .catch(() => console.error('Batch unsuccessful'));
 };
 
-seedManyUsers(100, 100);
\ No newline at end of file
+seedManyUsers(200, 10000);
```

Comment:
would be curious if this worked for a million if you tried :)

Reply:
lol. not brave enough to try it yet.